src/scripts/settings.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python
+from os import path
```

Comment:
Remove this file. #1 Contains it.

Reply:
I looked it and it does not contain it

Reply:
Before the PR has been closed it contained it. Leave it now.

Reply:
I suggest to use the class like container like in `constants.py`.